src/js/view/view.js

```diff
@@ -506,7 +506,6 @@ function View(_api, _model) {
         });
         if (_this.isSetup && aspectratio && !model.get('isFloating')) {
             style(_playerElement, getPlayerSizeStyles(model, model.get('width')));
-            _responsiveUpdate();
```

Comment:
it's important that we not trigger this when the player is not displayed. This should only be called after the transition is complete.